### ISSUE TITLE
model-admin doesn't detect models registered with the @admin.register decorator

### ISSUE THREAD

**Issue by stefanofusai** [kalekseev/django-extra-checks#37]
As the title says, it seems to me like model-admin doesn't detect models registered with the `@admin.register` decorator, as I'm getting lots of `(X012) The model is not registered in admin. [model-admin]` warnings for models that are, in fact, registered in  admin via the `@admin.register` decorator.

**Comment by kalekseev** [comment]
This rule just checks that models registred in `django.contrib.admin.sites.all_sites` (admin registry) so I'm not sure how that might not work. If you provide a repro I'll take a closer look at that.

**Comment by stefanofusai** [comment]
Sure thing, here's an example. Keep in mind that this seems to happen with all my models

The model:
```
class Currency(models.Model):
    code = models.CharField(max_length=3, unique=True)

    class Meta:
        verbose_name_plural = "currencies"

    def __str__(self) -> str:
        return self.code
```
The admin class:
```
@admin.register(Currency)
class CurrencyAdmin(ModelAdmin):
    list_display = ("id", "code")
```


The output: `core.Currency: (X012) The model is not registered in admin. [model-admin]`

**Comment by kalekseev** [comment]
@stefanofusai we have similar case in test app and it doesn't report an error https://github.com/kalekseev/django-extra-checks/blob/master/tests/example/admin.py

**Comment by stefanofusai** [comment]
Looks like the solution was a simple one: `"extra_checks"` must be placed **after** `"django.contrib.admin"` in `INSTALLED_APPS`, otherwise `django.contrib.admin.sites.all_sites` will be an empty set.

Maybe consider updating the README?

**Comment by stefanofusai** [comment]
Proposed a fix in [https://github.com/kalekseev/django-extra-checks/pull/38](https://github.com/kalekseev/django-extra-checks/pull/38)

**Comment by kalekseev** [comment]
Merged, thank you!
